homeassistant/components/websocket_api/decorators.py

```diff
@@ -31,7 +31,9 @@ def async_response(
     @wraps(func)
     def schedule_handler(hass, connection, msg):
         """Schedule the handler."""
-        hass.async_create_task(_handle_async_response(func, hass, connection, msg))
+        # As the webserver is now started before the start
+        # event we do not want to block for websocket responders
+        hass.loop.create_task(_handle_async_response(func, hass, connection, msg))
```

Comment:
We should use `asyncio.create_task` now. It's recommended from Python 3.7.

Reply:
Addressed in https://github.com/home-assistant/core/pull/36263